spec/selectors/transaction/broadcast.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  getTransactionStatus,
+  currentTransactionFailed,
+  currentTransactionBroadcasting,
+  currentTransactionBroadcasted,
+  getCurrentTransactionStatus
+} from 'selectors/transaction';
+import TEST_STATE from './TestState.json';
```

Comment:
Redux state is not strictly serializable, so many things will probably not come out right by saving this as JSON. What you more than likely want to do is to run the reducer once to get the initial state, and then manually modify the object with values you want to test. Here are a few examples of that:

Reducer tests https://github.com/MyCryptoHQ/MyCrypto/blob/develop/spec/reducers/rates.spec.ts
Selector tests from a branch I have https://github.com/MyCryptoHQ/MyCrypto/blob/reselect/spec/selectors/config.spec.ts

Reply:
That's definitely the solution I'm looking for. Sorry for the confusion. I'll implement tomorrow.